AirCasting/Graph/Graph.swift

```diff
@@ -124,21 +168,36 @@ class MultiColorGridRenderer: YAxisRenderer {
 struct Graph: UIViewRepresentable {
     typealias UIViewType = UI_PollutionGraph
     
+    @ObservedObject var stream: MeasurementStreamEntity
     @ObservedObject var thresholds: SensorThreshold
     
     func makeUIView(context: Context) -> UI_PollutionGraph {
         UI_PollutionGraph()
     }
     
     func updateUIView(_ uiView: UI_PollutionGraph, context: Context) {
-       try? uiView.updateWith(thresholdValues: thresholds.rawThresholdsBinding.wrappedValue)
+        
+        try? uiView.updateWith(thresholdValues: thresholds.rawThresholdsBinding.wrappedValue)
+        
+        let entries = stream.measurements?.compactMap({ item -> ChartDataEntry? in
```

Comment:
You can use `stream.allMeasurements` without `guard let measurement = item as? MeasurementEntity else {                return nil  }`